src/bottom-sheet/bottom-sheet.ts

```diff
@@ -0,0 +1,210 @@
+import { html, LitElement, PropertyValues, nothing } from 'lit';
```

Comment:
`tap-bottom-sheet` itself (not dimmer element) show be inside and element with `overflow-y: hidden` because its height changes pages height unintentional which is not desired.